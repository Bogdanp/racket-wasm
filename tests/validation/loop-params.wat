(module
 (func
  (loop
   (param i32)
   (param i64)
   (br 0))))
