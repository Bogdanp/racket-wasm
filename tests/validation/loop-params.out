(#f . "arity error\n  expected: [i32 i64]\n  found: []\n  in: loop-params.wasm\n  offset: 29 (0x1d)\n  at: (instr:loop #<loc> (functype '(i64 i32) '()) (vector (instr:br #<loc> 0)))\n  within: (funcidx 0)")